index.bs

```diff
@@ -1435,10 +1435,23 @@ To <dfn>cleanup the session</dfn> given |session|:
 
 1. [=Close the WebSocket connections=] with |session|.
 
+1. If [=active sessions=] is [=list/empty=], [=cleanup remote end state=].
+
 1. Perform any implementation-specific cleanup steps.
 
 </div>
 
+<div algorithm>
+To <dfn>cleanup remote end state</dfn>.
+
+1. [=map/Clear=] the [=before request sent map=].
+
+1. Set the [=default cache bypass=] to false.
+
+1. [=set/Empty=] the [=navigable cache bypass set=].
+
+</div>
```

Comment:
For multiple sessions, it could be an issue to clear the flag + set when one session ends?

Reply:
I've read again, and I think it actually implies that we should run these steps when/if all the sessions are closed, so should be fine with multiple sessions, I guess.

Reply:
Oh right indeed, thanks for checking.